packages/runtime/src/client/query-utils.ts

```
@@ -168,31 +131,19 @@ export function getUniqueFields(schema: SchemaDef, model: string) {
-    data: any
-): Record<string, any> {
+export function getIdValues(schema: SchemaDef, model: string, data: any): Record<string, any> {
     const idFields = getIdFields(schema, model);
     if (!idFields) {
         throw new InternalError(`ID fields not defined for model "${model}"`);
     }
-    return idFields.reduce(
-        (acc, field) => ({ ...acc, [field]: data[field] }),
-        {}
-    );
+    return idFields.reduce((acc, field) => ({ ...acc, [field]: data[field] }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)